app/src/main/res/values/strings.xml

```diff
@@ -5,4 +5,17 @@
     <string name="back4app_server_url">https://parseapi.back4app.com/</string>
     <string name="back4app_app_id">DZcZ5t4E5OwIQLRyPNYvprBQyD1G0Oy5nQlqrtpz</string>
     <string name="back4app_client_key">rjp5DDCIPgdeIkkh5cbW5WlTD2SxObooqcDh5FZQ</string>
+
+    <!--spinner prompt-->
+    <string name="prompt">categories</string>
+
+    <!-- Categories array for spinner options -->
+    <array name="categories">
```

Comment:
the source of truth for the categories here would be best populated in Java - I think it would make sense to have these items in here as individual strings, then specify which items go in the dropdown in Java, fetching their strings from the xml file.

This move is for style purposes and will be necessary if you want to have customizable categories - but if you want to wait until you implement customizable categories to make that change, that's no prob and could be quicker!

Reply:
Yeah, I was planning on having customizable categories later on, I'll keep this in mind for when I change it!